### DISCUSSION TITLE
Configurable nested CLIENT span suppression

### DISCUSSION THREAD

**Discussion by iNikem** [open-telemetry/opentelemetry-java-instrumentation#3318] [Q&A]
Recently spec [has been relaxed](https://github.com/open-telemetry/opentelemetry-specification/pull/1738) and now allows for nested CLIENT spans. At the same time we have clients who does not like our current decision to suppress nested CLIENT spans and want to see all of them. As spec now allows it, should we remove this suppression or make it configurable?

**Comment by trask** [comment]
Sounds reasonable, let's put it under an experimental flag to start with until we get more user feedback and/or more clarity on modeling in #1747
